packages/quantic/force-app/main/default/lwc/quanticRefineToggle/quanticRefineToggle.js

```diff
@@ -40,7 +40,8 @@ export default class QuanticRefineToggle extends LightningElement {
   labels = {
     sortAndFilters,
     viewResults,
-    noFiltersAvailableForThisQuery
+    noFiltersAvailableForThisQuery,
```

Comment:
```suggestion
    noFilterAvailableForThisQuery,
```

Reply:
Why do you judge this to be a better name?

Reply:
My bad, I had a discussion with Anny and I learnt that in english we can use 'No' and plural together.